communications/rfc.rst

```diff
@@ -16,13 +16,32 @@ On this page:
 - :ref:`rfc-jira`.
 - :ref:`rfc-cases`.
 - :ref:`rfc-discussion`.
-- :ref:`rfc-exceptions`.
 - :ref:`rfc-formalities`.
 
 .. seealso::
 
    :doc:`rfd` has a similar name, but has to do with convening discussions across DM teams.
 
+.. _rfc-cases:
+
+Standard procedures that require an RFC
+=======================================
+
+The following procedures **always** require an RFC:
+
+- :doc:`/stack/adding-a-new-package`.
+- :doc:`/stack/packaging-third-party-eups-dependencies`, or with Conda.
```

Comment:
Does this mean adding a new package to the scipipe conda env (and presumably any reference conda env we adopt for non-science pipelines -- including qserv?).  I suppose I need to follow that link because I don't think ccb cares about someone packaging an eups third party for testing, we care about adding a new third-party EUPS product as a dependency.

Reply:
> Does this mean adding a new package to the scipipe conda env (and presumably any reference conda env we adopt for non-science pipelines -- including qserv?)

Per RFC-537, yes, this requires an RFC (although that RFC has not yet been implemented).

We might want to consider whether that's still appropriate in light of recent changes to how we manage third parties, but it is the letter of the law as it now stands.